modules/starfish/2.0/starfish2.smk

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env snakemake
```

Comment:
Why is this snakefile named starfish2.smk ?

Reply:
Ooh, I did that to make sure the cookie cutter didn't overwrite the exisiting Starfish 1.0 snakefile. I'll drop the 2.